.circleci/config.yml

```diff
@@ -2,14 +2,20 @@ version: 2.1
 
 configuration:
   - cache_key: &cache_key v7-{{ checksum "src/xcode/Gemfile.lock" }}
-  - xcode_version: &xcode_version 12.1.0
+  - xcode_version: &xcode_version 12.2.0
 
 commands:
   install-tools:
     steps:
       - run:
           name: Install swiftlint
-          command: brew install swiftlint
+          command: |
+            if command -v swiftlint &> /dev/null
```

Comment:
Just out of curiosity why so complicated? I thought the CircleCI machines don't have SwiftLint installed?